sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.0.4 (2020-05-05)
+- Updated version of `azure-messaging-eventhubs` to `5.1.0`.
```

Comment:
Should the version of Storage be updated as well?

Reply:
Discussed offline with @alzimmermsft and @JimSuplizio. This change needs to wait until Identity and Storage modules are released.